workspace-manager-clienttests/src/main/resources/uploadlists/CompressDirectoryToTerraKernelK8S.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "CompressDirectoryToBucket",
+  "description": "Compress results directory to the terra-kernel-k8s-testrunner-results bucket",
+  "uploaderServiceAccountFile": "testrunner-iam-sa.json",
+  "uploadScripts": [
+    {
+      "name": "CompressDirectoryToBucket",
+      "description": "Save the compressed results directory to a bucket.",
+      "parameters": ["gs://terra-kernel-k8s-testrunner-results"]
```

Comment:
does this overwrite pre-existing files in the bucket, or is there any guarantee of uniqueness?

Reply:
No it won't overwrite preexisting file. Each test run generates a unique id which is part of the test results archive that pushed over to GCS.